example-plugins/patient_portal_care_team_widget/handlers/care_team_widget_handler.py

```diff
@@ -0,0 +1,53 @@
+from canvas_sdk.effects import Effect
+from canvas_sdk.effects.widgets import PortalWidget
+from canvas_sdk.events import EventType
+from canvas_sdk.handlers.base import BaseHandler
+from canvas_sdk.templates import render_to_string
+from canvas_sdk.v1.data.care_team import CareTeamMembershipStatus
+from canvas_sdk.v1.data.patient import Patient
+
+
+# Inherit from BaseHandler to properly get registered for events
+class CareTeamWidgetHandler(BaseHandler):
+    """The CareTeamWidgetHandler class is responsible for handling the care team widget event."""
+
+    # Name the event type you wish to run in response to
+    RESPONDS_TO = EventType.Name(EventType.PATIENT_PORTAL__WIDGET_CONFIGURATION)
+
+    def compute(self) -> list[Effect]:
+        """This method gets called when an event of the type RESPONDS_TO is fired."""
+        patient = Patient.objects.get(id=self.target)
+        patient_care_team = patient.care_team_memberships.filter(status=CareTeamMembershipStatus.ACTIVE)
```

Comment:
Given you're just using the patient to go through to the care team, you can skip one query and go directly to CareTeamMembership:

```suggestion
        patient_care_team = CareTeamMembership.objects.filter(patient__id=self.target, status=CareTeamMembershipStatus.ACTIVE)
```